src/Blazored.Toast.TestExtensions/BUnitToastTestExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Blazored.Toast.Services;
+using Blazored.Toast.TestExtensions;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Bunit
+{
+    [ExcludeFromCodeCoverage]
+    public static class BUnitToastTestExtensions
+    {
+        public static InMemoryToastService AddBlazoredToast(this TestContext context)
+        {
+            if (context is null)
+                throw new ArgumentNullException(nameof(context));
+
+            context.Services
+                .AddSingleton<IToastService, InMemoryToastService>();
+
+            return (InMemoryToastService)context.Services.GetService<IToastService>();
```

Comment:
Dont do it like this. As soon as you call `GetService`, the TestServiceProvider goes from being a service collection to a service provider, and no more services can be registered. Instead, do this:

```suggestion
            var toastService = new InMemoryToastService();
            context.Services.AddSingleton<IToastService>(toastService);
            return toastService;
```